Source/WebCore/page/scrolling/ScrollingStatePluginHostingNode.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2019-2023 Apple Inc. All rights reserved.
```

Comment:
2023

Reply:
These are all right since they're copy-pastes (this was already my adjusted version :) )